packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -1319,7 +1319,7 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
         currency
       )
       yield put(A.fetchCrossBorderLimitsSuccess(limitsResponse))
-    } catch (e: any) {
+    } catch (e) {
       yield put(A.fetchCrossBorderLimitsFailure(e))
```

Comment:
So this `fetchCrossBorderLimitsFailure` has to accept `unknown`, right?

Reply:
I usually do:

const error = errorHandler(e) // <= string